### ISSUE TITLE
Custom sidebar

### ISSUE THREAD

**Issue by egoist** [egoist/docute#24]
some people may want to custom their sidebar with pages instead of using navbar.

we can allow this by:

```js
self.$config = {
  sidebarNav: [
    // exactly same usage as `nav` but display in sidebar
    {title: 'About me', path: '/'}
  ]
}
```

And replace TOC with this.

**Comment by jasondavis** [comment]
It would be nice to be ale to override the current sidebar with a custom one.  

Docsify allows you to override there sidebar with a custom markdown file `_sidebar.md`

Some info here https://docsify.js.org/#/#sidebar

**Comment by egoist** [comment]
@jasondavis I might add an option to disable `TOC` in sidebar, and allow you to use plugin to register component in sidebar. btw, it does not make much sense to write sidebar in an external markdown file, `config.js` would be enough.

**Comment by egoist** [comment]
The `toc: false` option has landed in `v1.8.0`, and now we can also register component to the start or end of sidebar, you can simply add a plugin to fill custom contents in sidebar:

```js
registerComponent('sidebar:start', {
  template: 'custom content'
})
```

**Comment by egoist** [comment]
A plugin like this might be useful:

```js
self.config = {
  plugins: [
    customSidebarToc([
      {title: }
    ])
  ]
}
```
